force-app/main/default/classes/CON_ContactMerge_TEST2.cls

```diff
@@ -839,7 +836,7 @@ private class CON_ContactMerge_TEST2 {
             insert objectsToInsert;
             
             List<Contact> contactsToMerge = [
-                SELECT Id, AccountId, FirstName, LastName, MailingStreet, Description , Primary_Affiliation__c
+                SELECT Id, AccountId, FirstName, LastName, MailingStreet, Description , Primary_Affiliation__c, MailingCity
```

Comment:
And here. Just curious to learn a little bit more about why these query changes were needed.